sycl/test-e2e/Basic/launch_queries/max_num_work_groups.cpp

```diff
@@ -198,19 +207,20 @@ int test_max_num_work_groups(sycl::queue &q, const sycl::device &dev) {
     return 1;
   }
 
-  return 0;
+  return result;
```

Comment:
I'm not sure I understand the need for the `result` variable. In what case would we reach this and it not being `1`?